test/Microsoft.ML.Tests/Scenarios/PipelineApi/SimpleTrainAndPredict.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Data;
+using Microsoft.ML.Models;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Trainers;
+using Microsoft.ML.Transforms;
+using Xunit;
+
+namespace Microsoft.ML.Tests.Scenarios.PipelineApi
+{
+    public partial class PipelineApiScenarioTests
+    {
+        /// <summary>
+        /// Start with a dataset in a text file. Run text featurization on text values. 
+        /// Train a linear model over that. (I am thinking sentiment classification.) 
+        /// Out of the result, produce some structure over which you can get predictions programmatically 
+        /// (e.g., the prediction does not happen over a file as it did during training).
+        /// </summary>
+        [Fact]
+        void SimpleTrainAndPredict()
+        {
+            var dataPath = GetDataPath(SentimentDataPath);
+            var testDataPath = GetDataPath(SentimentDataPath);
+            var pipeline = new LearningPipeline();
+
+            pipeline.Add(MakeSentimentTextLoader(dataPath));
+
+            pipeline.Add(MakeSentimentTextTransform());
+
+            pipeline.Add(new FastTreeBinaryClassifier() { NumLeaves = 5, NumTrees = 5, MinDocumentsInLeafs = 2 });
+
+            pipeline.Add(new PredictedLabelColumnOriginalValueConverter() { PredictedLabelColumn = "PredictedLabel" });
+            var model = pipeline.Train<SentimentData, SentimentPrediction>();
+            var testData = MakeSentimentTextLoader(testDataPath);
```

Comment:
>MakeSentimentTextLoader [](start = 27, length = 23)

yuck

Reply:
rename testData to testPipe

---
In reply to: [210042025](https://github.com/dotnet/machinelearning/pull/677#discussion_r210042025) [](ancestors = 210042025)